spec/javascripts/this_is_my_org_spec.js

```diff
@@ -21,6 +21,11 @@ describe('This is my Organisation button', function() {
         menu = $('#menuLogin');
         toggle = $('#toggle_link');
         register = $('#registerForm');
+        timelapse = function() {
+          setTimeout(function() {
+            x = 5;
```

Comment:
Nothing, shouldn't be there ;). I was experimenting with stopping time in Jasmine specs. Deleting it.